src/pandroid/app/build.gradle.kts

```diff
@@ -39,4 +39,5 @@ dependencies {
     implementation("androidx.appcompat:appcompat:1.6.1")
     implementation("com.google.android.material:material:1.8.0")
     implementation("androidx.constraintlayout:constraintlayout:2.1.4")
+    implementation("com.google.code.gson:gson:2.10.1")
```

Comment:
gson seems unused for now. PRs should only contain code that is relevant, not code for future changes.